src/componentprocessor/index.ts

```diff
@@ -175,12 +185,15 @@ async function processXmlTree(
  * @param node Xml node with fields
  * @return The fields parsed as ComponentField
  */
-function getFields(node: XmlDocument): ComponentField {
+function processInterface(
```

Comment:
Mind updating the JSDoc when you get a second?

Reply:
done!